contracts/senders/aave-v2-migrator/main.sol

```diff
@@ -112,6 +112,26 @@ contract MigrateResolver is LiquidityResolver {
             "invalid-length"
         );
 
+        for (uint i = 0; i < _data.supplyTokens.length; i++) {
+            address _token = _data.supplyTokens[i];
+            for (uint j = 0; j < _data.supplyTokens.length; j++) {
+                if (j != i) {
+                    require(j != i, "token-repeated");
+                }
+            }
+            require(_token != wethAddr, "should-be-eth-address");
```

Comment:
Over here, we are converting into ETH => WETH: https://github.com/Instadapp/dsa-polygon-migration/blob/20a9e4fc6dc20e4983b70901a9a3fddef46e2c05/contracts/senders/aave-v2-connector/main.sol#L35-L36

Reply:
Oops! Let me set the limit for `ethAddr` then. Also, in connector, we'll need to also add for `borrowTokens`

And on Polygon does it treat WETH & ETH as 2 different tokens?

Reply:
On Polygon, there's only WETH since there's no ETH on Polygon